packages/editor/src/browser/editor-command.ts

```diff
@@ -297,25 +295,23 @@ export class EditorCommandContribution implements CommandContribution {
         }
         const reopenWithEncodingPick = { label: 'Reopen with Encoding', value: 'reopen' };
         const saveWithEncodingPick = { label: 'Save with Encoding', value: 'save' };
-        const actionItems: QuickPickItem<string>[] = [
+        const actionItems: Array<QuickPickValue<string>> = [
```

Comment:
why cahnge to Array<> from []

Reply:
I wanted to all arrays in file to be `Array<>` - it's the same as `[]` - but to me it looks more like a type then `aaa[]`.

Fixed